MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet private var errorView: ErrorView?
```

Comment:
Following the same logic as for MVP solution I recommend making `errorView` getter public and setter private.

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView?
```